docs/ref/checks.txt

```diff
@@ -239,6 +239,8 @@ Related Fields
 * **fields.E338**: The intermediary model ``<through model>`` has no field
   ``<field name>``.
 * **fields.E339**: ``<model>.<field name>`` is not a foreign key to ``<model>``.
+* **fields.E340**: The field's intermediary table name clashes with
```

Comment:
I was referring to the difference between "Field is using a table that has already been registered by" (code) and "The field's intermediary table name clashes with" (docs).